src/patternfly/components/OptionsMenu/options-menu.scss

```
@@ -75,7 +75,7 @@
   --pf-c-options-menu__group-title--PaddingBottom: var(--pf-c-options-menu__menu-item--PaddingBottom);
   --pf-c-options-menu__group-title--PaddingLeft: var(--pf-c-options-menu__menu-item--PaddingLeft);
   --pf-c-options-menu__group-title--FontSize: var(--pf-global--FontSize--sm);
```

should this be `--xs`?

```suggestion
  --pf-c-options-menu__group-title--FontSize: var(--pf-global--FontSize--xs);
```